moVirt/src/main/java/org/ovirt/mobile/movirt/ui/dashboard/maps/VmStatusMap.java

```diff
@@ -0,0 +1,44 @@
+package org.ovirt.mobile.movirt.ui.dashboard.maps;
+
+import org.ovirt.mobile.movirt.model.Vm;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by suomiy on 5/18/16.
+ */
+public enum VmStatusMap {
+    // dashboard position depends on the order
+    WARNING(Vm.Status.POWERING_UP, Vm.Status.PAUSED, Vm.Status.MIGRATING_FROM, Vm.Status.MIGRATING_TO, Vm.Status.MIGRATING, Vm.Status.UNKNOWN,
+            Vm.Status.WAIT_FOR_LAUNCH, Vm.Status.SUSPENDED, Vm.Status.POWERING_DOWN, Vm.Status.UNASSIGNED),
```

Comment:
- WAIT_FOR_LAUNCH should be in down
- migration_from, migrating_to, UP, migrating, powering_down should be in up